### ISSUE TITLE
`TrendsChart` and/or `LineChart` components with a11y issues

### ISSUE THREAD

**Issue by userquin** [npmx-dev/npmx.dev#1348]
We've a few a11y issues (go to compare page, add 2 packages and compare, then run LightHouse):
- inputs without labels (adding aria-label attribute to those inputs should be fine)
- increase size to at least 40px at bottom slider (touch targets do not have sufficient size or spacing)

**Comment by shuuji3** [comment]
@graphieros FYI: If you use Google Chrome (or maybe other chromium-based browser), you can run Lighthouse a11y test on DevTool and can see the recommendations (and referecne links) to fix detected issues.

ref. [Introduction to Lighthouse  |  Chrome for Developers](https://developer.chrome.com/docs/lighthouse/overview)

<img width="1300" height="1204" alt="screenshot of google chrome, left side shows comparison page, sight side shows light house result in devtool" src="https://github.com/user-attachments/assets/84dfdf47-00d4-4dc4-a58a-fcedfbba10c4" />
